Jenkinsfile

```diff
@@ -0,0 +1,21 @@
+library 'jenkins-ptcs-library@0.6.1'
+
+def project = 'ipa-log-app'
```

Comment:
project name differs from actual repository quite lot, is there some reason for this?

Reply:
The repo is about the same project (Log App) but with changed implementation,before it was a .NET app - now it's Grafana. There is already a folder in gcr named  'ipa-log-app' , so I kept the name.